src/common/plugin/pluginState.ts

```diff
@@ -1,11 +1,12 @@
 // Typescript translation from original code in edge-currency-bitcoin
 
+import { asObject } from 'cleaners'
 import { navigateDisklet } from 'disklet'
 import { EdgeIo, EdgeLog } from 'edge-core-js/types'
 import { makeMemlet, Memlet } from 'memlet'
 
 import { UtxoEngineState } from '../utxobased/engine/makeUtxoEngineState'
-import { ServerCache, ServerInfo } from './serverCache'
+import { asServerInfo, ServerCache, ServerInfo } from './serverCache'
 
 const InfoServer = 'https://info1.edge.app/v1'
 const FixCurrencyCode = (currencyCode: string): string => {
```

Comment:
Remove `FixCurrencyCode` because we're not using electrum servers anymore.

Reply:
Either we do all the work for the server-list task in this PR or we do the server-list work in it's own PR and rebase this PR onto that one.